ts/components/MemberListItem.tsx

```diff
@@ -20,12 +18,60 @@ const AvatarItem = (props: { memberPubkey: string; isAdmin: boolean }) => {
   );
 };
 
+const StyledSessionMemberItem = styled.button<{
+  inMentions?: boolean;
+  zombie?: boolean;
+  selected?: boolean;
+}>`
+  cursor: pointer;
+  flex-shrink: 0;
+  flex-grow: 1;
+  font-family: var(--font-default);
+  padding: 0px var(--margins-sm);
+  height: ${props => (props.inMentions ? '40px' : '50px')};
+  display: flex;
+
+  align-items: center;
+
+  justify-content: space-between;
+  transition: var(--default-duration);
+
+  opacity: ${props => (props.zombie ? 0.5 : 1)};
+
+  :not(:last-child) {
+    border-bottom: var(--border-session);
+  }
+
+  background-color: ${props =>
+    props.selected ? 'var(--color-conversation-item-selected) !important' : null};
```

Comment:
I think you can use `&&` here and not return null?

Reply:
the reason I am kind of used to put that syntax is because sometimes I treat the value as a boolean, and I should.
Like here: https://stackoverflow.com/a/53048160

Reply:
updated

Reply:
Cool